controllers/entities.js

```diff
@@ -30,9 +30,10 @@ exports.create = function(req, res, next) {
     if (result.errors > 0) return next(result.first_error);
     return result.changes[0].new_val;
   })
-  .then(entity =>
-    fanout.resolvePermissions(entity.id, req.body.permissions)
-    .then(() => res.send(entity)))
+  .then(entity => fanout.resolvePermissions(entity.id, req.body.permissions)
+    .then(() => r.table('entities').get(entity.id))
+    .then(entity => res.send(entity))
+  )
   .then(next)
```

Comment:
not part of your code, but i think this should be `.then(next.bind(null, null))`? otherwise the next callback will be invoked as an error if something is returned.